### PR TITLE
OCPBUGS#35997: Fix admission plugin url in upstream

### PR DESCRIPTION
**Pull request by opayne1**
Version(s):

Issue:
https://issues.redhat.com/browse/OCPBUGS-35997

Link to docs preview:

QE review:
- [ ] QE has approved this change
Just a link fix- QE not needed.

Additional information:

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ Topics:
   Distros: openshift-enterprise
 - Name: Admission plugins
   File: admission-plug-ins
-  Distros: openshift-enterprise,openshift-aro
+  Distros: openshift-enterprise,openshift-aro, openshift-origin
```
Comment (ShaunaDiaz) [inline _topic_maps/_topic_map.yml:108]:
```suggestion
  Distros: openshift-enterprise,openshift-aro,openshift-origin
```
Doesn't look like there should be a space.

Comment (opayne1) [reply _topic_maps/_topic_map.yml:108]: Whoops! Thank you!

**Review by bergerhoffer — COMMENTED** [review]
Actual change lgtm. Just making sure that admission plugins are actually available in OKD (that there wasn't a reason that this assembly wasn't included in OKD)?

**Comment by bergerhoffer** [comment]
Sending back since you said over slack you could merge it, you just needed a verification!

**Pull request merged**